kubectl-fdb/cmd/remove_process_group.go

```diff
@@ -105,21 +109,36 @@ kubectl fdb -n default remove process-group -c cluster --remove-all-failed
 	return cmd
 }
 
-// replaceProcessGroups adds process groups to the removal list of the cluster
-func replaceProcessGroups(kubeClient client.Client, clusterName string, ids []string, namespace string, withExclusion bool, wait bool, removeAllFailed bool, useProcessGroupID bool) error {
+// replaceProcessGroups adds process groups to the removal list of their respective clusters
+// if a clusterName is specified, it will ONLY do so for the specified cluster
+func replaceProcessGroups(kubeClient client.Client, clusterName string, ids []string, namespace string, clusterLabel string, withExclusion bool, wait bool, removeAllFailed bool, useProcessGroupID bool) error {
 	if len(ids) == 0 && !removeAllFailed {
 		return nil
 	}
 
-	cluster, err := loadCluster(kubeClient, namespace, clusterName)
+	// cross-cluster logic: given a list of Pod names, we can look up the FDB clusters by pod label, and work across clusters
+	if !useProcessGroupID && clusterName == "" {
+		pgsByCluster, err := fetchProcessGroupsCrossCluster(kubeClient, namespace, clusterLabel, ids...)
+		if err != nil {
+			return err
+		}
+		for cluster, pgs := range pgsByCluster {
+			// TODO check over this error logic
```

Comment:
leaving this as-is (abort entirely / return err when we get it, which is only when the user does not confirm or when a patch fails) would match the original logic, but I see the case for skipping of some clusters and not others, and the user can choose to abort the process if they choose to after seeing errors.